config/manager/manager.yaml

```diff
@@ -62,7 +62,7 @@ spec:
         - name: KATA_MONITOR_IMAGE
           value: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-monitor:latest
         - name: SANDBOXED_CONTAINERS_EXTENSION
-          value: kata-containers
+          value: sandboxed-containers
```

Comment:
This too looks like a local modification, I guess it shouldn't be part of this PR?

Reply:
Right